website/docs/docs/get-started/getting-started/getting-set-up/setting-up-databricks.md

```diff
@@ -15,9 +15,13 @@ This project will walk you through:
 
 - Setting up a Databricks account
 - Loading training data into your Databricks account
-- Configuring a SQL endpoint in Databricks
+- Configuring a SQL warehouse (formerly SQL endpoint) in Databricks
 - Connecting dbt Cloud and Databricks
 
+:::info Note 📌 
+Databricks SQL Endpoints' name has changed and is now called [**SQL Warehouses**](https://docs.databricks.com/sql/admin/sql-endpoints.html). 
```

Comment:
Looks like Databricks doesn't capitalize this so changing here for consistency with the rest of the doc

```suggestion
Databricks SQL Endpoints' name has changed and is now called [**SQL warehouses**](https://docs.databricks.com/sql/admin/sql-endpoints.html). 
```